components/Hero.tsx

```diff
@@ -9,19 +9,21 @@ export default function Hero() {
 
   return (
     <div className="homepage-img -mt-20 flex grow items-end justify-center bg-cover bg-fixed bg-[-10em] md:bg-[0]">
-      <div className="flex w-screen flex-col justify-center bg-gradient-to-b from-back/0 to-back/100 p-10 font-cbrush text-white">
-        <h1 className="text-center text-6xl sm:text-7xl lg:text-8xl">
+      <div className="flex w-screen flex-col justify-center bg-gradient-to-b from-transparent via-[#2a3b61d5] to-[#2A3B61] px-5 pt-5 font-cbrush text-white sm:p-10">
```

Comment:
This is the color matched to the figma but not the same as the background used on the rest of the site

Reply:
@GBKS is this color a one off or do we need to use this throughout the app? If its a one-off you can resolve this comment @benalleng! 😁

Reply:
@GBKS Just want to make sure this comment doesn't fall through the cracks! Mind taking a look when you get a chance?

Reply:
This pertains specifically to [Preview 1](https://saving-satoshi-r3v1op6fb-saving-satoshi.vercel.app/en)

Reply:
Hey Ben, kevin reviewed and approved. If you resolve this conversation I can merge.

Reply:
It's a one-off (sorry). In certain places it looks just tons better when colors match the images (like using a dark tone from the image instead of just black).